pubspec.yaml

```diff
@@ -6,4 +6,6 @@ environment:
   sdk: '>=1.0.0 <2.0.0'
 dependencies:
   browser: '>=0.10.0 <0.11.0'
-  svg_pan_zoom: '>=0.0.0 <0.1.0'
+  svg_pan_zoom:
+    git:
+      url: git://github.com/marianamarasoiu/svg_pan_zoom.git
```

Comment:
Please can you add a terminating line break, otherwise the file is tedious to work with via command line tools.